pyproject.toml

```diff
@@ -51,7 +51,7 @@ ignore = [
     "B019",
     "B023",
     "B028", # No explicit `stacklevel` keyword argument found
-    "B904", # Migrate from TRY200
+    "B904", # Re-raised error without specifying the cause via the from keyword
```

Comment:
Leaving it in the ignore list just disables. We need to explicitly select it, not ingore it.

Reply:
I didn't see they were in different lists. Thanks!